R/app.R

```diff
@@ -33,22 +33,24 @@ epiworldRenv <- function() {
 }
 
 #' @export
+#' @param custom_models_path Optional path to custom model files.
```

Comment:
Let's add a paragraph in the manual describing what this parameter do. Usually, you would do something like the following:

```suggestion
#' @param custom_models_path Optional path to custom model files (see details).
```

And then, in the details section

```r
#' @details
#' When `custom_models_path` is specified, the function will do ABC...
#' Particularly, the custom path expects R files and optionally md files with ....
```

And also, include an example:

```r
#' @examples
#' # Suppose we have the following R file:
#' ...
```

Reply:
Done.